packages/apps/plugins/plugin-deck/src/components/DeckLayout.tsx

```diff
@@ -53,39 +56,19 @@ export type DeckLayoutProps = {
   flatDeck?: boolean;
   toasts: ToastSchema[];
   onDismissToast: (id: string) => void;
-  location: Location;
+  layoutParts: LayoutParts;
   attention: Attention;
+  // TODO(Zan): Deprecate slots.
```

Comment:
how would those things be controlled?

Reply:
@thure What's the new version of this, variants and css variables?